src/main/java/be/seeseemelk/mockbukkit/block/data/BlockDataMock.java

```diff
@@ -452,9 +458,19 @@ else if (Tag.WALL_SIGNS.isTagged(material))
 		}
 		else if (Tag.BUTTONS.isTagged(material))
 		{
-			return new SwitchMock(material);
+			return new ButtonMock(material);
 		}
 		return null;
 	}
 
+	public ItemInteractionResult simulateUseItemOn(BlockStateMock blockStateMock, Location location, ItemStackMock itemStack, PlayerMock playerMock, EquipmentSlot equipmentSlot, BlockFace clickedFace)
+	{
+		return ItemInteractionResult.PASS_TO_DEFAULT_BLOCK_INTERACTION;
+	}
+
+	public InteractionResult simulateUseWithoutItem(BlockStateMock blockState, Location location, PlayerMock playerMock, BlockFace clickedFace)
+	{
+		return InteractionResult.PASS;
+	}
```

Comment:
Might be good to make this @ApiStatus.Internal?

Otherwise, if you want these to be part of the api, could you comment them?